x-pack/legacy/plugins/ml/public/application/explorer/actions/load_explorer_data.ts

```diff
@@ -143,10 +193,22 @@ export function loadExplorerData(state: ExplorerState) {
     tap(explorerService.setViewBySwimlaneLoading),
```

Comment:
Yes, the observables before that load all kinds of stuff but not the `viewBy` swimlanes because they depend on the overall swimlane data. So after among other things the overall data is loaded, the full-screen-loading page gets removed and we show the populated page except the with this `tap` the view-by swimlanes remain loading.